test/hades/mentorships/mentorships_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule Hades.MentorshipsTest do
+  use Hades.DataCase
+
+  alias Hades.Mentorships
+
+  describe "mentors" do
+    alias Hades.Mentorships.Mentor
+
+    @valid_attrs %{is_active: true, max_mentorships: 2, skill_areas: ["Backend"]}
+    @update_attrs %{is_active: false, max_mentorships: 3, skill_areas: ["Backend", "DevOps"]}
+    @invalid_attrs %{is_active: nil, max_mentorships: nil, skill_areas: []}
+    @invalid_skill_areas_attrs %{is_active: false, max_mentorships: 3, skill_areas: ["Bad", "Skills"]}
+
+    def mentor_fixture(attrs \\ %{}) do
+      {:ok, mentor} =
+        attrs
+        |> Enum.into(@valid_attrs)
+        |> Mentorships.create_mentor()
+
+      mentor
+    end
+
+    test "list_mentors/0 returns all mentors" do
+      mentor = mentor_fixture()
```

Comment:
I think is better have mentor inside setup and use it wherever we need passing it through a map. i.e.: `%{mentor: mentor}`

Reply:
Good idea, i think that will look better. Will do in a different PR!